### ISSUE TITLE
Remove AI Logo Maker by Fiverr in Site Settings

### ISSUE THREAD

**Issue by mcrowell** [elementor/elementor#19823]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The issue is this new ad for Fiverr is causing clutter on the backend settings and can cause confusion to our clients.

We don't want our clients to come across a sales pitch for Fiverr logo design if they paid money for us to do a professional branding package. 

### Describe the solution you'd like

Give us an option to atleast turn off this ad for Fiverr.

**Comment by fabianenzensperger** [comment]
Elementor do stupid things that nobody need..

**Comment by sf-steve** [comment]
Is this real, i presumed the OP had their site compromised. If they are actually adding fiverr ads to a $1k/year agency product, thats absolutely shocking. @mcrowell where are you seeing this please

**Comment by hitsgee** [comment]
Please add an option to disable. I charge for this service and do not want to advertise they could have got it for free.

**Comment by mcrowell** [comment]
> Is this real, i presumed the OP had their site compromised. If they are actually adding fiverr ads to a $1k/year agency product, thats absolutely shocking. @mcrowell where are you seeing this please

@sf-steve This was added in 3.7.5.

**Comment by m1ga** [comment]
This one?
![Screenshot_20220915_191212](https://user-images.githubusercontent.com/4334997/190468231-54e7f0f8-610a-46cc-82d6-fbcc5833aa81.png)

Add a class to your theme
```
.elementor-control-custom_logo_promotion {
  /* fiverr ad in site-settings */
  display: none;
}
```

or use https://github.com/m1ga/elementor_just_free

**Comment by alloutsidethebox** [comment]
This is garbage, does not belong at all. 
I vote that feature off the island.

**Comment by hyperjit** [comment]
In a world where customers (I don't say, users, I say customers who are paying for your product) are begging to unbloat the core and add only the most essential features, it is surprisingly awful how this team finds every piece of crap to add and bloat the plugin even further. Considering how much they are really listening to their customers these days (which is zero), hoping this feature would be subtracted is a distant dream.

**Comment by portrayaloflife** [comment]
Agreed, this needs to go

**Comment by DigitalBerry** [comment]
Just in case anyone at Elementor is unsure on my opinion on this....

**### Elementor grow the fuck up and take this shit out!**

Clear enough for you...

**Comment by hitsgee** [comment]
What next Adverts on loading screens? Amazon affiliate links on the settings pages.

**Comment by jamieburchell** [comment]
Elementor: Welcome to Your Future
https://user-images.githubusercontent.com/3270426/190632989-9bd372e0-6acc-4155-85d5-1ca4519f9297.mp4

**Comment by mcrowell** [comment]
If anyone wants to manually remove this Fiverr ad from the setting panel you can remove the following code from the file:
(\elementor\core\kits\documents\tabs\settings-site-identity.php)


--------------------------------------------------------------
		$this->add_control(
			'custom_logo_promotion',
			[
				'type' => Controls_Manager::RAW_HTML,
				'raw' => '<p>' . esc_html__( 'Add a logo to display on your website. Don\'t have one yet? Create a professional logo with Fiverr\'s logo maker.', 'elementor' ) .
					'</p>' .
					sprintf(
						'<a target="_blank" class="elementor-button e-logo-maker" href="https://go.elementor.com/site-settings-logo-maker/">%s</a>',
						esc_html__( 'Create a Logo in Minutes', 'elementor' )
					),
				'condition' => [
					'site_logo[url]' => '',
				],
				'content_classes' => 'elementor-panel-alert elementor-panel-alert-success',
			]
		);
--------------------------------------------------------------

**Comment by toineenzo** [comment]
wow, this horrible. If they at least integrated the logo maker in Elementor, that would be nice. But this is such a use less feature, with just an affilliate link for Elementor to earn some money...

**Comment by bolded1** [comment]
And now lets get serious...  If they added an affiliate link there.. It SCARES me about the actual position Elementor is at the moment and whats its future.. I literally checked it 2 times to believe it.
 For me this is the new low for page builders..

**Comment by m1ga** [comment]
As mentioned above: use the CSS to hide it. 
But honestly I don't mind it there, I don't have to click on it :shrug: And I use the default WP customizer to change the logo and site name :smile: I knew that it was there because I saw the PR but I had to search to find it to look at it. Not having it is of course better but honestly: I have plugins that have more annoying ads in them :rofl:

**Comment by bolded1** [comment]
I will not use css to hide it. I demand this to be gone. i pay for this software that doesnt include ads. 
If we dont act to this i am afraid what the next update will bring.

**Comment by m1ga** [comment]
ah ok, didn't know that it was in the Pro version too. I only use the free version! Ok it shouldn't be in it if you have a pro licence

**Comment by alloutsidethebox** [comment]
This is a huge slap in our faces and needs to go asap. I won't renew or recommend to clients until this is gone. 
SO RUDE TO TREAT YOUR CUSTOMERS THIS WAY!

**Comment by KAYLEEJVR** [comment]
What an absolute joke. I’ve been searching for the best solution to make a column clickable yet you release a fiver logo ad feature. Ads on a paid plug-in?

**Comment by pauserratgutierrez** [comment]
OMG, put whatever you want to Elementor Free, but not in a licence of Elementor Pro.

**Comment by bhambrabland** [comment]
Shocking, so much so, that I have posted about this in the community.

Elementor, you need to pay attention now. Your loyal customer base is NOT happy. The pros are not happy. We pay a lot of money for this platform. What is going on?

https://www.facebook.com/groups/Elementors/permalink/5204945936297817/

**Comment by hitsgee** [comment]
Seems they have listed and a pull request has been added removing this stupid feature. https://github.com/elementor/elementor/pull/19860

**Comment by fabianenzensperger** [comment]
The update just came and it was removed. Thanks Elementor for the quick reaction, just fended off another shitstorm. Incidentally, you can avoid something like this in the future if you ask your community about such a controversial "feature" beforehand. This saves you a lot of hassle and work.

@shilo-ey, @avivu, @arielk, etc.

**Comment by pauserratgutierrez** [comment]
Thanks elementor for listening to the community. That's the fastest reaction to fix something that the community requests ever done by you, in my opinion. Thanks again.

**Comment by mcanayok** [comment]
Such an amazing feature, removed. I'm super sad.

Elementor Team, please get your shit straight if you don't want to end up like WP Bakery.

Oh and can't wait the day Gutenberg fully takes over.

**Comment by VerdiH** [comment]
<img width="529" alt="image" src="https://user-images.githubusercontent.com/20367733/191437859-f336e25a-7e89-4ff5-8dd4-206704f08f30.png">

https://www.facebook.com/groups/Elementors/permalink/5208376069288137

We removed it. Thank you for your feedback!!

**Comment by avivu** [comment]
This issue has been resolved in Elementor v3.7.7

Feel free to update

Thanks!
